dev.vencord.Vesktop.yml

```diff
@@ -0,0 +1,68 @@
+app-id: dev.vencord.Vesktop
+
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+
+command: startvesktop
+separate-locales: false
+
+finish-args:
+  - --socket=pulseaudio
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications # Discord push notifs
+  - --talk-name=org.kde.StatusNotifierWatcher # Tray functionalities on KDE
+  - --device=all # Needed for GPU acceleration and the webcam
+  - --filesystem=xdg-videos:ro
+  - --filesystem=xdg-pictures:ro
+  - --filesystem=xdg-download # This and the above two are used for drag-n-drop, and download managing
+  - --filesystem=xdg-run/pipewire-0 # Pipewire interfacing
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons # This is used to show the correct cursor on Wayland
+
+modules:
+  - name: minerals
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 minerals/startvesktop /app/bin/startvesktop
+      - install -D minerals/dev.vencord.Vesktop.desktop -t /app/share/applications/
+      - install -D dev.vencord.Vesktop.metainfo.xml -t /app/share/metainfo/
+    sources:
+      - type: dir
+        path: .
```

Comment:
Don't use `type: dir`

Reply:
I'm confused on how I'm meant to import minerals if I don't use `type: dir`.

Reply:
also this should be with the main module. It doesn't even have a relatable name make it confusing.

Reply:
The name was based off of the [ArmCord flatpak](https://github.com/flathub/xyz.armcord.ArmCord). I'm aware all of it is probably no longer up to standard but I'm trying to learn from how they do things alongside the existing guides.

Reply:
> I'm confused on how I'm meant to import minerals if I don't use `type: dir`.

type: file, path: minerals/filename

Reply:
Yep just worked it out